src/main/java/com/fasterxml/jackson/databind/DeserializationContext.java

```diff
@@ -465,28 +465,18 @@ public final boolean hasSomeOfFeatures(int featureMask) {
     public final JsonParser getParser() { return _parser; }
 
     public final Object findInjectableValue(Object valueId,
-            BeanProperty forProperty, Object beanInstance)
+            BeanProperty forProperty, Object beanInstance, Boolean optional)
```

Comment:
Yeah this makes sense, just need that deprecation thingy for backwards-compatibility.